CardScan/CardScan/Classes/ScanConfiguration.swift

```diff
@@ -1,6 +1,12 @@
 import Foundation
 
+@objc public enum ScanPerformance: Int {
+    case fastScan
+    case accurateScan
+}
+
 @objc public class ScanConfiguration: NSObject {
     @objc public var runOnOldDevices = false
     @objc public var setPreviouslyDeniedDevicesAsIncompatible = false
+    @objc public static var scanPerformancePriority: ScanPerformance = .fastScan
```

Comment:
I wouldn't make this a static configuration variable -- it should be a member variable on `SimpleScanViewController` that defaults to `fast` and people can pick `accurate` on a per viewcontroller basis. For example, if someone is using both scan and verify they might want different configurations for the two.